docs/sidebars.js

```diff
@@ -245,7 +245,7 @@ const sidebars = {
           ],
         },
         {
-          label: "Build Uniswap with Portals",
+          label: "Build a Uniswap Integration with Portals",
```

Comment:
```suggestion
          label: "Build an Aztec Connect style Uniswap Integration with Portals",
```

Reply:
i dont want to use aztec connect in the title because it gatekeeps people who dont know what that is, and there'll be more & more of them as time goes by. wdyt?

Reply:
hmm okay rename to "Swap on L1 Uniswap from L2 with Portals"